### PR TITLE
refactor: remove simplane calls from pfd

### PR DESCRIPTION
**Pull request by Saschl**
Fixes #[issue_no]

## Additional context

Discord username (if different from GitHub):

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/PFD/shared/PFDSimvarPublisher.tsx [hunk 1]
```diff
@@ -115,6 +115,14 @@ export interface PFDSimvars {
     xtk: number;
     ldevRequestLeft: boolean;
     ldevRequestRight: boolean;
+    baroMode: number;
+    baroUnit: number;
+    baroPressure: number;
```
Comment (tracernz) [inline src/instruments/src/PFD/shared/PFDSimvarPublisher.tsx:120]: Maybe a good idea to split these into left/right but reading the same A vars so we don't need to edit the PFD code later (just switch the A vars for L vars).

**Review by 2hwk — APPROVED** [review]

**Comment by 2hwk** [comment]
May want to look at the merge conflicts?

**Comment by 2hwk** [comment]
Will be impacted by the potential future merge freeze. https://github.com/flybywiresim/aircraft/pull/8466 will be required to be merged into this PR at some point.

**Comment by lukecologne** [comment]
This has been implemented with #7587

**Pull request closed without merging**